server/status/runtime.go

```diff
@@ -170,6 +179,18 @@ func (rsr *RuntimeStatSampler) SampleEnvironment() {
 		log.Errorf(context.TODO(), "unable to get cpu usage: %v", err)
 	}
 
+	fds := gosigar.ProcFDUsage{}
+	if err := fds.Get(pid); err != nil {
```

Comment:
because `FDUsage` returns system-wide usage (it looks up `/proc/sys/fs/file-nr`) whereas `ProcFDUsage` looks up that process' usage and limits.